.github/wiki/Getting-Started.md

```diff
@@ -11,5 +13,5 @@
 Run the Rails app:
 
 ```sh
-make dev
-``
\ No newline at end of file
+bundle exec make dev
```

Comment:
I would recommend editing the "MakeFile" so that each command called by the make file will use `bundle exec` and that you will have a shorter command to run each time 😇 

```suggestion
make dev
```

Reply:
Good suggestion, I updated. 👍